### ISSUE TITLE
Test datasets for revised DCAT vocabulary

### ISSUE THREAD

**Issue by dr-shorthair** [w3c/dxwg#137]
Any revisions to the DCAT vocabulary must not break existing DCAT implementations. 
The exact criteria for evaluating this will need to be decided. 
Test data, and test services, should be found to allow assessment. 

Probably start with the datasets identified in the original [implementation report](https://www.w3.org/2011/gld/wiki/DCAT_Implementations).

**Comment by akuckartz** [comment]
:+1:

**Comment by davebrowning** [comment]
Handle as part of implementation evidence gathering, or at least in parallel with it?

**Comment by andrea-perego** [comment]
I wonder whether we should move this to future work, until we receive feedback about backward compatibility issues (I may be wrong but, to my knowledge, we have not received any so far).

**Comment by riccardoAlbertoni** [comment]
+1 to move it on future work

**Comment by andrea-perego** [comment]
Done.

**Comment by andrea-perego** [comment]
I wonder whether we should close this issue. DCAT 2 is out and we are receiving feedback on it. I don't we need a test suite at this stage.

**Comment by andrea-perego** [comment]
Noting no objections, I'm closing this issue.
